packages/scatterplot/src/ScatterPlotItem.js

```diff
@@ -21,18 +21,33 @@ const ScatterPlotItem = ({
     onMouseMove,
     onMouseLeave,
     onClick,
-}) => (
-    <circle
-        cx={x}
-        cy={y}
-        r={size / 2}
-        fill={color}
-        onMouseEnter={onMouseEnter}
-        onMouseMove={onMouseMove}
-        onMouseLeave={onMouseLeave}
-        onClick={onClick}
-    />
-)
+    symbol,
+    point,
+}) =>
+    symbol ? (
+        React.createElement(symbol, {
+            x,
+            y,
+            size,
+            color,
+            onMouseEnter,
+            onMouseMove,
+            onMouseLeave,
+            onClick,
+            point,
+        })
+    ) : (
+        <circle
```

Comment:
This could be the default symbol instead of having to repeat all the props

Reply:
I put the default symbol as default prop, is this what you wanted ? let me know.